apps/docs/content/pages/getting-started.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: Getting started
+description: Getting started with Workleap Design Tokens
+---
+
+This guide will help you get started with Hopper UI. Hopper UI is a Design System that provides a collection of components, tokens, and icons to help you build applications.
+
+## Quick Install
+
+To use Hopper in your project, run one of these commands in your terminal.
+
+<PackageInstallation library="components @hopper-ui/icons @hopper-ui/tokens @hopper-ui/styled-system" mod="prod" />
+
+After installing Hopper UI packages, you must setup a Provider at the root of your application. This is typically in your _index.tsx_ or _app.tsx_, but can vary based on your specific setup.
+
+```jsx
+import { HopperProvider } from "@hopper-ui/components";
+
+<HopperProvider colorScheme="light">
+    <Button>Hello World</Button>
+</HopperProvider>
+```
+
+<Callout>Hopper is compatible with React 18. Not using React, or using an older version than 18? You can still benefit from Hopper, read on to discover how.</Callout>
+
+## Detailed Installation
+
+### Components
+
+Hopper Components are React Comtponents, uf you are using another framework, or React Native, you should use the [components tokens](/tokens/getting-started/introduction) directly when developping your own components.
+
+#### Installation
+
+Run one of the following commands in your terminal to install the Hopper Components package.
```

Comment:
There is only one command under. I understand you are refering to either npm/pnpm/yarn, but there is 1 command, for 3 different tools. I would phrase it like 

```
Run the following command in your terminal to install the Hopper packages.
```